DfE.FindInformationAcademiesTrusts/Pages/Trusts/Contacts.cshtml

```diff
@@ -50,37 +41,64 @@
           <dt class="govuk-summary-list__key">
             Accounting officer
           </dt>
-          <dd class="govuk-summary-list__value">
-            <p class="govuk-body">Tyler Welch</p>
-            <p class="govuk-body">
-              <a class="govuk-link" href="#">Tyler.Welch@abbeylaneacademiestrust.co.uk</a>
-            </p>
-          </dd>
+          @{ DisplayContact(Model.AccountingOfficer); }
         </div>
         <div class="govuk-summary-list__row">
           <dt class="govuk-summary-list__key">
             Chair of trustees
           </dt>
-          <dd class="govuk-summary-list__value">
-            <p class="govuk-body">Courtney Pacocha</p>
-            <p class="govuk-body">
-              <a class="govuk-link" href="#">Courtney.Pacocha@abbeylaneacademiestrust.co.uk</a>
-            </p>
-          </dd>
+          @{ DisplayContact(Model.ChairOfTrustees); }
         </div>
         <div class="govuk-summary-list__row">
           <dt class="govuk-summary-list__key">
             Chief financial officer
           </dt>
-          <dd class="govuk-summary-list__value">
-            <p class="govuk-body">Lowell Hoppe</p>
-            <p class="govuk-body">
-              <a class="govuk-link" href="#">Lowell.Hoppe@abbeylaneacademiestrust.co.uk</a>
-            </p>
-          </dd>
+          @{ DisplayContact(Model.ChiefFinancialOfficer); }
         </div>
 
       </dl>
     </div>
   </div>
 </section>
+
+@functions {
+
+  private void DisplayContact(Person? contactToDisplay)
+  {
+    <dd class="govuk-summary-list__value">
+
+      @if (contactToDisplay is null)
```

Comment:
If `contactToDisplay is null` then currently we will be displaying the `@ContactsModel.ContactInformationNotAvailableMessage` twice - here and on line 88
We should only be showing it once

Reply:
line 88 populates the email field with the message if no contact data is found as per scenario 2 in the ac

Scenario 2
GIVEN a user is on the 'Contacts' page
WHEN all of the contact data is unavailable for a contact
THEN a message of 'no contact information available' should be shown in each field where no data is available

Reply:
Ah, I think the word "field" is a bit ambiguous here - I read "field" as the whole name/email box but I see that the same word is used on the other ACs when referring to just the name or email.
We should clarify with the designer/PM to see what the desired behaviour is

Reply:
sorted james wants just 1 message displayed